### ISSUE TITLE
The console does not announce the command result to the screen reader

### ISSUE THREAD

**Issue by bazzdg** [Microsoft/monaco-editor#970]
**Actual**:
On the console when user type any command, the result displays on the console for the particular console, the screen reader does not announce the command result.

**Expected**:
On the console when user type any command, the result displays. The screen reader should announce the command result. 

**User Impact**:
Users who are dependent on screen reader will find difficulty if the screen reader does not announce the command result.

**Test Environment**:
OS: Win 10
OS version: 1709 
OS build: 16299.98
Browser : Edge

**MAS Reference**:
MAS 1.3.1 - Info and relationship

**Comment by alexandrudima** [comment]
@bazzdg What product are you testing? That is outside of the monaco editor.
